src/main/java/pl/isa/javaers/RegisterUser/PasswordEdit.java

```diff
@@ -0,0 +1,16 @@
+package pl.isa.javaers.RegisterUser;
+
+import java.io.File;
+
+public class PasswordEdit {
```

Comment:
Logicznie nic nam nie daje zrobienie edycji hasła w osobnej klasie. Gdyby to było w jednej, to mógłbyś wyodrębnić powtarzające się operacje na plikach do osobnych funkcji.